### ISSUE TITLE
Add Dialogue Policy

### ISSUE THREAD

**Issue by kbalog** [iai-group/DialogueKit#156]
next_intent() goes here

**Comment by WerLaj** [comment]
@NoB0 @kbalog We could have a separate class `DialoguePolicy` that implements the `next_dialogue_act()` method that predicts the next dialogue act given the last user `utterance` and `dialogue_history`. This method is used in `receive_utterance()` in the agent to generate the agent's response. We probably want to predict the next dialogue act not only the intent, as intent annotations may be needed for generating the agent's response.

**Comment by kbalog** [comment]
@WerLaj Yes to creating a new `DialoguePolicy`class. I suppose in most cases we'd want to predict next dialogue act, but that'll likely involve next intent prediction as a first substep.
